packages/stark-ui/src/modules/route-search/components/route-search.component.spec.ts

```diff
@@ -0,0 +1,153 @@
+/* tslint:disable:completed-docs no-commented-code prefer-const*/
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { RouteEntry } from "@nationalbankbelgium/stark-ui";
+import { TranslateModule } from "@ngx-translate/core";
+import { Component, NO_ERRORS_SCHEMA, ViewChild } from "@angular/core";
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
+import { NoopAnimationsModule } from "@angular/platform-browser/animations";
+import { RouteSearchComponent } from "./route-search.component";
+import { MatButtonModule } from "@angular/material/button";
+import { MatInputModule } from "@angular/material/input";
+import { MatFormFieldModule } from "@angular/material/form-field";
+import { MatAutocompleteModule } from "@angular/material/autocomplete";
+import { MatSelectModule } from "@angular/material/select";
+import { MatOptionModule } from "@angular/material/core";
+import { STARK_ROUTING_SERVICE, StarkRoutingService } from "@nationalbankbelgium/stark-core";
+import { MockStarkRoutingService } from "@nationalbankbelgium/stark-core/testing";
+import { CommonModule } from "@angular/common";
+import Spy = jasmine.Spy;
+@Component({
+	selector: `host-component`,
+	template: `
+		<route-search [options]="optionsList"></route-search>
+	`
+})
+class TestHostComponent {
+	@ViewChild(RouteSearchComponent)
+	public routeSearchComponent: RouteSearchComponent;
+	public optionsList: RouteEntry[];
+}
+
+/* tslint:disable:no-big-function */
+fdescribe("RouteSearchComponent", () => {
+	let component: RouteSearchComponent;
+	let hostComponent: TestHostComponent;
+	let hostFixture: ComponentFixture<TestHostComponent>;
+
+	const mockRoutingService: StarkRoutingService = new MockStarkRoutingService();
+
+	const optionsList: RouteEntry[] = [
+		{
+			path: "path 1",
+			url: "/url1"
```

Comment:
Weird that we can specify the "url" property on a `RouteEntry` because such interface doesn't have that property 😕

Reply:
Yup, I had to push my branch directly after having done a lot of changes that haven't been tested yet that's why everything is so messed up. I had to work on the app-data urgently so we can make a release by tuesday, I'll continue with the route search right after that ;)